Detections/SecurityEvent/GroupCreatedAddedToPrivlegeGroup_1h.yaml

```diff
@@ -35,15 +36,56 @@ query: |
   | where TargetSid !in ("S-1-5-32-555")
   | where TargetSid matches regex WellKnownLocalSID or TargetSid matches regex WellKnownGroupSID
   | project GroupAddTime = TimeGenerated, GroupAddEventID = EventID, GroupAddActivity = Activity, GroupAddComputer = Computer, GroupAddTargetAccount = TargetAccount, 
-  GroupAddTargetSid = TargetSid, GroupAddSubjectAccount = SubjectAccount, GroupAddSubjectUserSid = SubjectUserSid, GroupSid = MemberSid;
-  let GroupCreated = SecurityEvent
+  GroupAddTargetSid = TargetSid, GroupAddSubjectAccount = SubjectAccount, GroupAddSubjectUserSid = SubjectUserSid, GroupSid = MemberSid
+  ),
+  (
+  WindowsEvent 
+  // 4728 - A member was added to a security-enabled global group
+  // 4732 - A member was added to a security-enabled local group
+  // 4756 - A member was added to a security-enabled universal group  
+  | where EventID in ("4728", "4732", "4756")  and not(EventData has "S-1-5-32-555")
+  | extend SubjectUserSid = tostring(EventData.SubjectUserSid)
+  | extend Account = strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
+  | extend AccountType=case(Account endswith "$", "Machine", iff(SubjectUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", iff(isempty(SubjectUserSid), "", "User")))
```

Comment:
Sorry. Just noticed this now. The case/iff combination here is not needed and awkward. You probably wanted the below. Worth replacing backwards.

case 
    (Account endswith "$" or SubjectUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", 
     isempty(SubjectUserSid), "",
     "User")